frontend/src/layouts/PublicLayout.scss

```diff
@@ -21,6 +21,23 @@
   min-height: 0;
 }
 
+//for displaying a background image on the login page
+.App-content-background {
```

Comment:
why not put this on Login.scss?

Reply:
The login form is the only place this background is going to be used right?

Reply:
I tried that, but App-content was the only place I could get the image to fit the entire screen.

Reply:
you should be able to set the height of the .unauth class in Login.scss doing something like this:
```
@import '../../variables.scss';
height: calc(100vh - #{$footer-height} - #{$header-height} - #{$navbar-height});
```
you'll probably need to change the margin on the unauth class as well to padding.
That should make the .unauth element the same size as app-content

Reply:
Got it!